### DISCUSSION TITLE
Seq 4.2 -> 5.1 -> 2022 -> 2023 Upgrade

### DISCUSSION THREAD

**Discussion by PaulFarry** [datalust/seq-tickets#2059] [General]
I've been following the process for upgrading from 4.2-> 2023, and I've got that all under control, but the timing for waiting for completion is tripping me up. (I know I need to allow time for the upgrade, but ultimately I won't be doing the deployments)

I'd like to ensure  that if the deployment (Using Octopus) is performed too soon from 5.1 -> 2022 that if there hasn't been enough delay between 2022 and 2023 then I will abort the upgrade. Presently if you upgrade too soon, there is an error but it's only visible if you do `seq run` from the command prompt (The seq service writes just a "Service Started" followed by a "Service Stopped" to `eventvwr`).

If there is a `seq HELP ` that I can run to verify that the upgrade has completed before attempting the 2023 upgrade then I can fail my upgrade.

Any tips would be very helpful.

**Comment by nblumhardt** [comment]
Hi @PaulFarry, thanks for dropping us a line.

If possible, it's best to do these steps individually, and allow at least some manual inspection/burn-in time between them. The 5.1->2022.1 upgrade particularly can take a while.

At each step, if anything has gone wrong it'll be easier to resolve or rollback then, rather than continue upgrading and potentially complicate matters :-)

Dropping us a line via `support@` noting the days/times you're planning to perform the upgrade will also help us prepare, in case you end up needing a hand.

Hope this helps,
Nick
